docs/_articles/en/v3.x/salesforce-graph-engine/rules.md

```diff
@@ -163,6 +164,68 @@ Parameter explanation:
 
 The operation dereferences a null object and throws a NullPointerException. Review your code and add a null check.
 
+## MultipleMassSchemaLookupRule
+
+MultipleMassSchemaLookupRule is a path-based rule that detects scenarios where expensive schema lookups are made more than one time in a path and cause performance degradation. 
+
+These methods are identified by MultipleMassSchemaLookupRule.
+
+* `Schema.getGlobalDescribe()`
+* `Schema.describeSObjects(...)`
+
+Flagged lookups include:
+
+* Lookups within these types of loops: ForLoopStatement, ForEachLoopStatement, DoWhileStatement, and WhileLoopStatement
+* More than one invocation in a path 
+
+These common scenarios trigger a violation from MultipleMassSchemaLookupRule.
+* `Schema.getGlobalDescribe()` within a loop
+* `Schema.describeSObjects(...)` within a loop
+* `Schema.getGlobalDescribe()` preceding a `Schema.getGlobalDescribe()` or `Schema.describeSObjects(...)` method call anywhere in the path
+* `Schema.describeSObjects(...)` preceding a `Schema.describeSObjects(...)` or `Schema.getGlobalDescribe()` method call anywhere in the path
+
+### Definitions
+
+| Rule Component | Definition                                  																		|
+| ---------		 | ---------                                															  			|
+|**Source**		 |																													|
+|		 	  	 | `@AuraEnabled`-annotated methods     																			|
+|				 |`@InvocableMethod`-annotated methods																			  	|
+|     			 | `@NamespaceAccessible`-annotated methods 																		|
+| 				 |`@RemoteAction`-annotated methods																				  	|
+|				 |Any method returning a `PageReference` object																	  	|
+|				 |`public`-scoped methods on Visualforce Controllers																|
+|				 |`global`-scoped methods on any class																			  	|
+|				 |`Messaging.InboundEmailResult handleInboundEmail()` methods on implementations of `Messaging.InboundEmailHandler`	|
+|				 |Any method targeted during invocation																				|
+| **Sink**		 | 																													|
+| 		  	  	 |`Schema.getGlobalDescribe()`													|
+|				 |`Schema.describeSObjects(...)`																										|
+| **Sanitizer**	 |				                                  														      		|
```

Comment:
We could probably skip "sanitizer" entirely. Unless there's a term for something that should be prevented. The "thing" to be prevented is making more than one mass schema lookup within the path.